src/transformers/generation_utils.py

```diff
@@ -478,6 +478,11 @@ def _expand_inputs_for_generation(
         if attention_mask is not None:
             model_kwargs["attention_mask"] = attention_mask.index_select(0, expanded_return_idx)
 
+        lm_attention_mask = model_kwargs.get("lm_attention_mask", None)
```

Comment:
(nit) - would stick to the same signature:

```suggestion
        if "lm_attention_mask" in model_kwargs:
            lm_attention_mask = model_kwargs["lm_attention_mask"]
```